README.md

```diff
@@ -6,50 +6,39 @@
 [![Tests](https://github.com/FREVA-CLINT/freva-nextgen/actions/workflows/ci_job.yml/badge.svg)](https://github.com/FREVA-CLINT/freva-nextgen/actions)
 [![Test-Coverage](https://codecov.io/github/FREVA-CLINT/freva-nextgen/branch/init/graph/badge.svg?token=dGhXxh7uP3)](https://codecov.io/github/FREVA-CLINT/freva-nextgen)
 
-This repository contains the *freva-rest services* defining rest endpoints
-that make up the freva server services as well as the client
-services that provide command line interfaces and python libraries for their
-rest service counterparts.
 
-## Table of Contents
+This is a multi-part repository it contains code for:
 
-- [Installation](#installation)
-- [Usage](#usage)
-- [Development Environment](#development-environment)
-- [Testing](#testing)
-- [License](#license)
+- The *freva-rest service* defining rest endpoints
+  that make up the freva server services
+- The *freva-client* library that provide command line interfaces and python
+  libraries for their rest service counterparts.
+- The *freva-data-loader-portal* that implements rules of how to open different
+  data sources and stream them via zarr.
 
-## Installation
+## Installation for development
 
-1. Make sure you have Python 3.11+ installed.
+1. Make sure you have Python 3.8+ installed.
 2. Clone this repository:
 
 ```console
-git clone git@github.com:FREVA-CLINT/freva-nextgen.git
+git clone --recursive git@github.com:FREVA-CLINT/freva-nextgen.git
 cd freva-nextgen
 ```
 
-3. Install flit:
-Flit is used as the build system for development purpose. Install flit
-into your current python environment:
+3. Install all components:
 
 ```console
-python3 -m pip install flit
+python -m pip install -e ./freva-rest[dev] -e ./freva-client -e ./freva-data-portal-worker[full]
 ```
 
-4. Install the rest-api:
+4. Generate a new pair of self signed certificates
```

Comment:
True

Reply:
Because i just tested it and i found it makes certs automatically without generating via this command in the mentioned lines, i asked ...

If you say so it's good from my side